frame/proxy/src/lib.rs

```diff
@@ -250,11 +250,23 @@ pub mod pallet {
 			proxy_type: T::ProxyType,
 			delay: T::BlockNumber,
 		) -> DispatchResult {
-			let who = ensure_signed(origin)?;
-			let delegate = T::Lookup::lookup(delegate)?;
-			Self::remove_proxy_delegate(&who, delegate, proxy_type, delay)
+			
+			Self::remove_all_proxy_delegates(&who, delegate, proxy_type, delay)
+		}
+
+		
+		#[pallet::weight(T::WeightInfo::remove_all_proxy_delegates(T::MaxProxies::get()))]
+		pub fn pub remove_all_proxy_delegates(origin: OriginFor<T>,
```

Comment:
`pub fn pub` What is that?

Reply:
Sorry for that. I fixed it. I was having issue with the rust analyser it was failing to build so this got missed. I will take care next time

Reply:
I don't see a new commit pushed to your PR fixing this issue?